pkg/monitor/azure/nsg/nsg.go

```diff
@@ -0,0 +1,176 @@
+package nsg
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"context"
+	"net/http"
+	"net/netip"
+	"strings"
+
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore/arm"
+	mgmtnetwork "github.com/Azure/azure-sdk-for-go/services/network/mgmt/2020-08-01/network"
+	"github.com/Azure/go-autorest/autorest"
+	"github.com/sirupsen/logrus"
+
+	"github.com/Azure/ARO-RP/pkg/api"
+	"github.com/Azure/ARO-RP/pkg/metrics"
+	"github.com/Azure/ARO-RP/pkg/util/azureclient"
+	"github.com/Azure/ARO-RP/pkg/util/azureclient/mgmt/network"
+)
+
+const (
+	DimSubnet              = "subnet"
+	DimVnet                = "vNet"
+	DimResourceGroup       = "resourcegroup"
+	DimSubscriptionID      = "subscriptionid"
+	DimSecurityGroup       = "networksecuritygroup"
```

Comment:
I recommend this naming convention to keep it consistent with the following 5 consts:
```suggestion
	DimNSG       = "networksecuritygroup"
```

Reply:
done